wagtail_streamfield_migration_toolkit/test/tests/test_migrations.py

```diff
@@ -0,0 +1,353 @@
+        )
+        instances[1] = factories.SampleModelFactory(
+            content__0__char1="Test char 1",
+            content__1__char1="Test char 2",
+            content__2__char2="Test char 3",
+        )
+        instances[2] = factories.SampleModelFactory(
+            content__0__char2="Test char 1",
+            content__1__char2="Test char 2",
+            content__2__char2="Test char 3",
+        )
+
+        cls.instances = {
+            instance.id: instance.content.raw_data for instance in instances
+        }
+
+    def test_migrate_stream_data(self):
+        self.apply_migration()
+
+        instances = models.SampleModel.objects.all().annotate(
+            raw_content=Cast(F("content"), JSONField())
+        )
+
+        for instance in instances:
+            prev_content = self.instances[instance.id]
+            for old_block, new_block in zip(prev_content, instance.raw_content):
+                self.assertEqual(old_block["id"], new_block["id"])
+                if old_block["type"] == "char1":
+                    self.assertEqual(new_block["type"], "renamed1")
+                else:
+                    self.assertEqual(old_block["type"], new_block["type"])
+
+
+class TestPage(BaseMigrationTest):
+    model_name = "SamplePage"
+
+    @classmethod
+    def setUpTestData(cls):
+        instances = [None for i in range(3)]
+        instances[0] = factories.SamplePageFactory(
+            content__0__char1="Test char 1",
+            content__1__char1="Test char 2",
+            content__2__char2="Test char 3",
+            content__3__char2="Test char 4",
+        )
+        instances[1] = factories.SamplePageFactory(
+            content__0__char1="Test char 1",
+            content__1__char1="Test char 2",
+            content__2__char2="Test char 3",
+        )
+        instances[2] = factories.SamplePageFactory(
+            content__0__char2="Test char 1",
+            content__1__char2="Test char 2",
+            content__2__char2="Test char 3",
+        )
+
+        cls.instances = {}
+        cls.revisions = {}
+        for instance in instances:
+            cls.instances[instance.id] = instance.content.raw_data
+            for i in range(5):
+                revision = instance.save_revision()
+                revision.created_at = datetime.datetime.now() - datetime.timedelta(
+                    days=(5 - i)
+                )
+                if i == 1:
+                    instance.live_revision = revision
+                    instance.save()
+            cls.revisions[instance.id] = list(instance.revisions.all().order_by("id"))
+
+    def test_migrate_stream_data(self):
+        self.apply_migration()
+
+        instances = models.SamplePage.objects.all().annotate(
+            raw_content=Cast(F("content"), JSONField())
+        )
+
+        for instance in instances:
+            prev_content = self.instances[instance.id]
+            for old_block, new_block in zip(prev_content, instance.raw_content):
```

Comment:
If this is going to be used as the standard assertion for all these tests, could we make it a method of the base class rather than making you write it out each time?